go/pkg/command/command.go

```diff
@@ -71,6 +71,17 @@ func (s SymlinkBehaviorType) String() string {
 	return fmt.Sprintf("InvalidSymlinkBehaviorType(%d)", s)
 }
 
+func SymlinkBehavior(s string) SymlinkBehaviorType {
```

Comment:
Nit: the name sounds more like a type. I'd use `ParseSymlinkBehavior` which is akin to `strconv.ParseInt`. `SymlinkBahaviorFromString` is another reasonable name which is akin to `uuid.FromString`.

Reply:
Ah, I've just noticed there is already a `symlinkBehaviorFromProto` in this file.